docs/docs/integrations/document_loaders/pebblo.ipynb

```diff
@@ -124,6 +124,36 @@
   {
    "cell_type": "markdown",
    "metadata": {},
+   "source": [
+    "### Anonymize the snippets to redact all PII details\n",
+    "\n",
+    "Set `anonymize_snippets` to `True` to anonymize all personally identifiable information (PII) from the snippets going into VectorDB and the generated reports."
```

Comment:
Is the recall on identifying PII 100%? If not might be worth a clarification or link to other documentation to set expectations.

Reply:
@ccurme I’ve added a note and a link to our docs. We’re planning to update it with more details soon. Please review.